kratos/geometries/geometry.h

```diff
@@ -3301,8 +3297,7 @@ class Geometry
 
         double detJ;
         Matrix Jinv(this->WorkingSpaceDimension(), this->WorkingSpaceDimension());
```

Comment:
Not related to the changes in here, but shouldn't this be
```suggestion
        Matrix Jinv(this->LocalSpaceDimension(), this->WorkingSpaceDimension());
```

Reply:
Nice catch!